components/motor/gpio/encoded.go

```diff
@@ -260,6 +260,15 @@ func (m *EncodedMotor) GoFor(ctx context.Context, rpm, revolutions float64, extr
 	ctx, done := m.opMgr.New(ctx)
 	defer done()
 
+	warning, err := checkSpeed(rpm, m.cfg.MaxRPM)
+	if warning != "" {
+		m.logger.CWarnf(ctx, warning)
+	}
+	if err != nil {
+		m.logger.CWarn(ctx, "motor speed requested is 0 rev_per_min")
```

Comment:
do we need this if we have the above warning?

Reply:
nope, good catch.